dataland-dummy-qa-service/DockerfileBase

```diff
@@ -0,0 +1,11 @@
+ARG DATALAND_GRADLE_BASE_VERSION
+FROM ghcr.io/d-fine/dataland/dataland_gradle_base:${DATALAND_GRADLE_BASE_VERSION} as builder
+
+WORKDIR /app
+COPY ./dataland-dummy-qa-service/ ./dataland-dummy-qa-service/
+COPY ./dataland-backend-utils/ ./dataland-backend-utils/
+COPY ./dataland-message-queue-utils/ ./dataland-message-queue-utils/
+COPY ./.git/ ./.git
+RUN ./gradlew dataland-dummy-qa-service:bootJar --no-daemon --stacktrace
+
+HEALTHCHECK --interval=30s --timeout=3s CMD wget -nv -O- -t 1 http://localhost:8080/qa/actuator/health | tee /dev/stderr | grep -q '^{"status":"UP"' || false
```

Comment:
missing newline, 
I think the healthcheck doesn't make sense here, as the base-image wil never be healthy.

Reply:
I thought healthcheck is just needed to be set somewhere. We also put it inside the dockerfilebase in the inound proxy. However, I did not have a deeper look into that. Will be checked.

Reply:
Now it's in the "final" images. So it's ok now.